### DISCUSSION TITLE
浅谈 Puma 的实现 · /rack-puma

### DISCUSSION THREAD

**Discussion by draveness** [draveness/blog-comments#514] [Announcements]
https://draveness.me/rack-puma

**Comment by howborn** [comment]
博主，用的什么工具画图哦。

**Comment by cryptocj** [comment]
请问这个负载均衡由 OS 层来做是哪儿体现的? 
是所有的 worker 都 IO.select 同样的 socket导致的吗?

**Comment by shaojunda** [comment]
> 如过当前任务不需要立即处理，

「如过」-> 如果

---

*2020-07-18 UPDATES：已修复*

**Comment by icbd** [comment]
如果 PUMA 支持了 Ractor, 还有必要选用集群模式吗?
